src/components/AppLayout/Sidebar/SafeHeader/index.tsx

```diff
@@ -60,15 +71,96 @@ const StyledIcon = styled(Icon)`
 const IconContainer = styled.div`
   width: 100px;
   display: flex;
-  padding: 4px 0;
+  gap: 8px;
   justify-content: space-evenly;
+  margin: 14px 0;
 `
 const StyledButton = styled(Button)`
   &&.MuiButton-root {
+    width: 100%;
+    height: 38px;
     padding: 0 12px;
   }
-  *:first-child {
-    margin: 0 4px 0 0;
+`
+
+const StyledExplorerButton = styled(ExplorerButton)`
+  border-radius: 5px;
+  width: 32px;
+  height: 32px;
+  background-color: ${background};
+
+  & span {
+    transition: background-color 0.2s ease-in-out;
+    border-radius: 5px;
+    width: 32px;
+    height: 32px;
+    background-color: ${background};
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  & .icon-color {
+    fill: ${secondary};
+  }
+
+  &:hover span {
+    background-color: ${primaryLite};
+  }
```

Comment:
This code seems to be repeated in the next 2 styles as well. Can we abstract it?

Reply:
I'd approach this with caution. Style abstaction can be very confusing.